[CI] Rolling-on-humble build is broken

https://github.com/ros-controls/ros2_control_ci/actions/runs/25128549610/job/73648269163?pr=761
```
  /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/src/ros-controls/mujoco_ros2_control/mujoco_ros2_control_plugins/src/external_wrench_plugin.cpp: In member function ‘void mujoco_ros2_control_plugins::ExternalWrenchPlugin::publish_markers()’:
  /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/src/ros-controls/mujoco_ros2_control/mujoco_ros2_control_plugins/src/external_wrench_plugin.cpp:285:16: error: ‘class realtime_tools::RealtimePublisher<visualization_msgs::msg::MarkerArray_<std::allocator<void> > >’ has no member named ‘tryPublish’; did you mean ‘try_publish’?
    285 |   marker_pub_->tryPublish(msg);
        |                ^~~~~~~~~~
        |                try_publish
```

Introduced with #165

Conditional on RCLCPP_VERSION_MAJOR
https://github.com/ros-controls/mujoco_ros2_control/blob/dc158b4700d1d675a5f636b6b63e876db693bccd/mujoco_ros2_control_plugins/src/external_wrench_plugin.cpp#L282-L286

should be changed to ROS_DISTRO_HUMBLE like here
https://github.com/ros-controls/mujoco_ros2_control/blob/dc158b4700d1d675a5f636b6b63e876db693bccd/mujoco_ros2_control/src/mujoco_system_interface.cpp#L1504-L1508


As an alternative, I can deactivate the build of mujoco_ros2_control in the compatibilty build.

--- a/mujoco_ros2_control_plugins/CMakeLists.txt
+++ b/mujoco_ros2_control_plugins/CMakeLists.txt
@@ -48,6 +48,9 @@ target_link_libraries(mujoco_ros2_control_plugins PUBLIC
   ${visualization_msgs_TARGETS}
   ${mujoco_ros2_control_msgs_TARGETS}
 )
+target_compile_definitions(mujoco_ros2_control_plugins PRIVATE
+  REALTIME_TOOLS_VERSION_MAJOR=${realtime_tools_VERSION_MAJOR}
+)
 
 # Install the plugin library
 install(TARGETS mujoco_ros2_control_plugins

--- a/mujoco_ros2_control_plugins/src/external_wrench_plugin.cpp
+++ b/mujoco_ros2_control_plugins/src/external_wrench_plugin.cpp
@@ -17,7 +17,6 @@
 #include <algorithm>
 #include <string>
 
-#include <rclcpp/version.h>
 #include <geometry_msgs/msg/point.hpp>
 #include <pluginlib/class_list_macros.hpp>
 #include <visualization_msgs/msg/marker.hpp>
@@ -279,7 +278,7 @@ void ExternalWrenchPlugin::publish_markers()
 {
   MarkerArray msg;
   publish_markers(msg);
-#if RCLCPP_VERSION_MAJOR > 16
+#if REALTIME_TOOLS_VERSION_MAJOR > 2
   marker_pub_->try_publish(msg);
 #else
   marker_pub_->tryPublish(msg);

